src/main/java/yevhenii/lostfilmdemo/quartz/JobConfig.java

```diff
@@ -0,0 +1,45 @@
+package yevhenii.lostfilmdemo.quartz;
+
+import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
+import org.quartz.JobBuilder;
+import org.quartz.JobDetail;
+import org.quartz.Trigger;
+import org.quartz.TriggerBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import static org.quartz.SimpleScheduleBuilder.simpleSchedule;
+import static org.quartz.TriggerKey.triggerKey;
+
+@Log4j2
+@Configuration
+public class JobConfig {
+    @Getter
+    private static int jobCount = 1;
+    @Getter
+    private static int triggerCount = 1;
+
+    @Bean
+    public JobDetail getJobDetail() {
+        log.info("creating job");
+        return JobBuilder.newJob(LostfilmRssJob.class)
+                .storeDurably()
+                .withIdentity("lostfilm_job" + jobCount++, "lostfilmGroup")
```

Comment:
check this example how properly define identity https://github.com/kenshoo/fb-product-fetcher/blob/master/application/src/main/java/io/skai/fbproductfetcher/quartz/config/QuartzJobConfig.java#L19

No need adding any counters you did here